spec/features/constestants/index_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+RSpec.describe "Constestant Index Page" do
+    it "should show the names of all constants" do
+        challenge1 = Challenge.create!(theme: "Apartment Furnishings")
+        challenge2 = Challenge.create!(theme: "Fashion")
+
+        project1 = Project.create!(name: "Litfit", material: "Lamp Shade", challenge: challenge1)
+        project2 = Project.create!(name: "Rug Tuxedo", material: "Rug", challenge: challenge1)
+        project3 = Project.create!(name: "LeatherFeather" , material: "Leather", challenge: challenge2)
+
+        contestant1 = Contestant.create!(name: "Kentaro Kameyama")
+        contestant2 = Contestant.create!(name: "Jay McCarroll")
+
+        ContestantProject.create!(contestant: contestant1, project: project1)
+        ContestantProject.create!(contestant: contestant1, project: project2)
+        ContestantProject.create!(contestant: contestant2, project: project3)
+
+        visit "/contestants"
+
+        expect(page).to have_content("Kentaro Kameyama")
+        expect(page).to have_content("Projects: Litfit, Rug Tuxedo")
+        expect(page).to have_content("Jay McCarroll")
+        expect(page).to have_content("Projects: LeatherFeather")
```

Comment:
Good job testing multiple contestants and giving them a different number of projects. I think it's a good idea to always test the single case (they have 1 of something) and then a multiple (they have 2+ of something). This will ensure our formatting looks good for both cases.